ui/src/app/applications/components/application-create-panel/application-create-panel.tsx

```diff
@@ -120,15 +121,12 @@ export const ApplicationCreatePanel = (props: {
                 load={() =>
                     Promise.all([
                         services.projects.list().then(projects => projects.map(proj => proj.metadata.name).sort()),
-                        services.clusters
-                            .list()
-                            .then(clusters =>
-                                clusters.map(cluster => ({label: clusterTitle(cluster), value: cluster.server})).sort((first, second) => first.label.localeCompare(second.label))
-                            ),
+                        services.clusters.list().then(clusters => clusters.map(cluster => cluster.server).sort()),
```

Comment:
The `services.clusters.list()` sends API request, so we execute `GET /v1/api/clusters` twice unnecessary. Lets change code to execute`services.clusters.list()` once and use output to produce list of names and urls.